### ISSUE TITLE
Feature Toggle Config

### ISSUE THREAD

**Issue by jack-mil** [slushie0/qolify#3]
Is there any configuration where individual features can be toggled?
I like Fullbright and Zoom, but want to disable the hotbar number display. 
I don't see any config in ModMenu or config/ folder

No problem if this isn't implemented yet, just checking that I am not missing something

**Comment by slushie0** [comment]
No, there isn't any config yet though I plan to add this in the future. It will be via modmenu when it is added though.
